src/main/java/com/funnyboyroks/drawlib/renderer/ShapeRenderer.java

```diff
@@ -48,6 +47,13 @@ public void setColor(@NotNull Color color) {
         this.color = color;
     }
 
+    /**
+     * @param step_size The new step between two particles. Lower values can crash low-end devices
```

Comment:
Change the description of this parameter to be something more like "The new step between particles.  Lower values may cause server/client performance issues.  Higher values may make the lines hard to see.  Must be greater than 0."